src/promise-katas-1.js

```diff
@@ -33,72 +33,128 @@ Remember, a Promise can only resolve successfully, or reject when it fails to re
+const returnBob = () => { 
+  return new Promise((resolve) => {
+    resolve({ name: 'Bob' });
+  });
+};
 
 // 5 Create a function that returns a promise which returns an array with 4 strings of 'eggs', 'apples', 'milk' and 'bread' if it resolves succesfully
 // Example:
 // returnList() resolves to the value ['eggs', 'apples', 'milk', 'bread']
 
-const returnList = () => { };
+const returnList = () => { 
+  return new Promise((resolve) => {
+    resolve(['eggs', 'apples', 'milk', 'bread']);
+  });
+};
 
 // 6 Create a function that returns a promise which returns "An error occurred" if it rejects
 // Example:
 // anError() rejects to "An error occurred"
 
-const anError = () => { };
+const anError = () => { 
+  return new Promise((_, reject) => {
+    reject("An error occurred");
+  });
+};
 
 // 7 Create a function that returns a promise which returns 666 if it rejects
 // Example:
 // theNumberOfTheBeast() rejects to the value 666
 
-const theNumberOfTheBeast = () => { };
+const theNumberOfTheBeast = () => { 
+  return new Promise((_, reject) => {
+    reject(666);
+  });
+};
 
 // 8 Create a function that returns a promise which returns an object with a key of error and value of 500 if it rejects
 // Example:
 // internalServerError() rejects to { error: 500 }
 
-const internalServerError = () => { };
+const internalServerError = () => { 
+  return new Promise((_, reject) => {
+    reject({ error: 500 });
+  });
+};
 
 // 9 Create a function that returns a promise which resolves to "happy" if the parameter is greater than
 // or equal to 1 and rejects to "sad" if the value is less than 1
 // Example:
 // happySad(1) resolves to "happy"
 // happySad(0) rejects to "sad"
 
-const happySad = () => { };
+const happySad = (num) => { 
+  return new Promise((resolve, reject) => {
+    
+    if (num >= 1) {
+    resolve("happy");
+    } else {
+      reject("sad");
+    };
+  });
+};
 
 // 10 Create a function that returns a promise which resolves to "Yes. Luke, I am your father." if the
 // parameter is "Luke" and rejects to "Not your dad." if the value is anything else
 // Example:
 // amIYourFather() resolves to "Yes. Luke, I am your father."
 // amIYourFather() rejects to "Not your dad."
 
-const amIYourFather = () => { };
+const amIYourFather = (name) => { 
+  return new Promise((resolve, reject) => {
+    if (name === "Luke") {
+      resolve("Yes. Luke, I am your father.");
+      } else {
+        reject("Not your dad.");
+      };
+  });
+};
 
 // 11 Create a function that returns a promise. The promise should resolve to a new function that returns "My name is <input name>"
 // Example:
 // A call to myNameIs() resolves to a new function that takes one argument, name, and returns "My name is name" Eg (name) => `My name is ${name}`
 
-const myNameIs = () => { };
+const myNameIs = () => { 
+  return new Promise((resolve, reject) => {
+   resolve(test);
+  });
+};
 
+const test = (name) => {
+  return `My name is ${name}`;
+  };
```

Comment:
The closing bracket of the function should be at the same level as where the function was opened:
```
};
```